frame/democracy/src/lib.rs

```diff
@@ -192,6 +192,9 @@ const DEMOCRACY_ID: LockIdentifier = *b"democrac";
 /// NOTE: This is not enforced by any logic.
 pub const MAX_VETOERS: u32 = 100;
 
+/// The maximum number of public proposals that can exist at any time.
+pub const MAX_PROPOSALS: u32 = 1000;
+
```

Comment:
In general I would prefer these kinds of consts live as configurable from the pallet's Trait. The feedback we have heard many times from downstream users is the ability to configure their pallet and their weights for their needs and use-cases...

I wont block on it though.